### DISCUSSION TITLE
How do i conect the FanControl software with L-Conect3

### DISCUSSION THREAD

**Discussion by Laurinmuell** [Rem0o/FanControl.Releases#2516] [Q&A]
Hi 
Maby i am just stupid but i tried to add both of the Lian Li plugins but it is not working correctly. 
I was only able to add the zip file with both plugins into the programm with the plugin add option instead of adding a single one into the plugin folder. The programm did not acept a file put in manually into the folder. I deinstalled one of the add ons after i had to add both.
Inside the programm i can see now the 4 chanels of my controller and also the the speed of two chanels what should be my two fan blocks with 3 fans each what is good but i cant change the speed of the fans.
I also tried to close L-Conet 3 with taskmanager so it wouldn`t be able to controll the fans but it changed nothing. 
does anyone know what i am doing wrong ?
![FanControll_Overlay](https://github.com/Rem0o/FanControl.Releases/assets/170091917/9a024a8b-d437-4adf-9aa9-c220cafd9a8c)
![Plugin_Folder](https://github.com/Rem0o/FanControl.Releases/assets/170091917/bfd72331-63c9-4eed-ae70-a8ae8891bd3a)
![Device_Manager](https://github.com/Rem0o/FanControl.Releases/assets/170091917/6fcbd060-17d9-4ef5-960f-10226da99d87)

**Comment by Rem0o** [comment]
I believe there is some option in L-Connect, Motherboard RPM sync or something like that, it needs either to be on or off, can't remember, for the plugin to work.

**Comment by mcouture87** [comment]
I installed L-Connect to setup the number of fans and saved it to the controller.   I did not set L-Connect to "motherboard sync".   I then un-installed L-Connect.   I also uninstalled MSI Mystic Light (I have a MSI motherboard).   I am using OpenRGB to control my RGB so I am using the non-RGB plugin for FanControl.

I have FanControl and OpenRGB run on Windows startup and my fans are controlled fine by FanControl and my RGBs from my motherboard and my Lian Li controlled fans work just fine.

Long story short is that L-Connect is probably controlling the fans and you need to disable or un-install L-Connect -  If memory serves me, there is at least 3 services for L-Connect that you need to disable if you don't  want to un-install it.

**Reply by robvalkyrie** [reply]
Do I need to install the L-Connect software first? Just purchased new fans, will be installing tomorrow (with controller). My question I guess is do I need to even install the L-Connect software first or can I just install the Plugin?
